content/app_competition_rules.tex

```diff
@@ -71,17 +87,132 @@ \subsection{Winner and Rankings}
 
 \end{enumerate}
 
-\subsection{Champions Cup and Challenge Shield}
-\label{sec:twoCompetitions}
-In order to provide better matched games for teams of all abilities the RoboCup Standard Platform League shall be divided into two separate competitions: the Champions Cup for the strongest teams and the Challenge Shield for all other teams. Final assignment of teams to each competition occurs at RoboCup based on initial game performance.
+\subsubsection{Promotion Games}
+
+The bottom 2 teams of each group of the Champions Cup after the round-robin, along with all semi-finalist teams of the Challenge Shield knock-out competition proceed to the promotion games.
+
+There will be 4 promotion games. By random draw, each game will have one team from the Champions Cup and one team from the Challenge Shield. The winning team of the promotion game will be guaranteed a spot in the Champions Cup in the next RoboCup tournament. The loosing team of the promotion game, may still be placed in the Champions Cup in the next RoboCup tournament subject to available places and their Glicko ranking.
+
+
+\subsubsection{Champions Cup selection}
+
+Teams are selected for the Champions Cup as follows until all places are filled:
+\begin{enumerate}
+    \item All qualified teams that made the knockout stage of the previous year SPL RoboCup Champions Cup.
+    \item All qualified teams that won the promotion game  of the previous year SPL RoboCup Champions Cup.
+    \item The highest Glicko ranked qualified team, until all remaining places are filled.
+\end{enumerate}
+
+All remaining qualified teams are then selected for the Challenge Shield.
+
+
+\subsection{Challenge Shield Rule Changes}
+\label{sec:cs-rule-changes}
+
```

Comment:
My suggestion at this point would be to place the changes next to to corresponding rule in a box. This way, everybody sees that this rule differs whenbetween the two Leagues

Reply:
If the TC is happy with the changes then, this can be considered. However, for the revival, it is easier to simply see everything all at once rather than trawl the rest of the rules.

Reply:
I have now pushed a set of references (using todo notes for now) in the main body to where rules have been changed.